### PR TITLE
OCP 201 -  misc fixes

### PR DESCRIPTION
**Pull request by ShellyXueHan**
### changes:
- added a list of prerequisites for ocp201 lab
- some fixes/updates for resource management content to match upgraded cluster and BCGov specifics

### CODE REVIEW

File: openshift-201/README.md [hunk 1]
```diff
@@ -2,10 +2,18 @@
 Welcome to the OpenShift 201 Training Labs.  You will find all of the lab materials in this folder which accompany the 201 presentations found [here](https://app.mural.co/t/platformservices5977/m/platformservices5977/1648237994578/6d72899801ba0c9b04e4f120571621c188c92036?sender=u66de390c4d3da408f9803733).
 
 
+### Prerequisites:
+- completion of the OCP101 lab material. You can find the material hosted on [this GitHub Page](https://ocp101-labs-d8f105-tools.apps.silver.devops.gov.bc.ca/) or [source content here](https://github.com/BCDevOps/devops-platform-workshops/tree/master/101-lab/content).
+- oc CLI installed, find out [how to here](https://stackoverflow.developer.gov.bc.ca/questions/139)
+- an OpenShift project set created for you from Registry app, which you are an Admin of. You will either receive an email about this or during the OCP201 kick off.
```
Comment (mtspn) [inline openshift-201/README.md:8]:
```suggestion
- Completion of the OCP101 lab material. You can find the material hosted on [this GitHub Page](https://ocp101-labs-d8f105-tools.apps.silver.devops.gov.bc.ca/) or [source content here](https://github.com/BCDevOps/devops-platform-workshops/tree/master/101-lab/content).
- oc CLI installed, find out [how to here](https://stackoverflow.developer.gov.bc.ca/questions/139)
- An OpenShift project set created for you from Registry app, which you are an Admin of. You will either receive an email about this in advance of the course, or set this up during the OCP201 kick off.
```

File: openshift-201/README.md [hunk 2]
```diff
@@ -2,10 +2,18 @@
...
+
+
+### Lab topics:
+
 The following are the labs related to the 201 Courses
```
Comment (mtspn) [inline openshift-201/README.md:13]:
```suggestion
The Openshift 201 Lab is divided into the following topics: ```

File: openshift-201/resource-mgmt.md [hunk 3]
```diff
@@ -114,19 +114,19 @@ spec:
...
           command: ["/opt/rh/httpd24/root/usr/bin/ab"]
           args: ["-dSrk", "-c $(CONCURRENCY)", "-n $(REQUESTS)", "https://$(SERVICE_HOST):$(SERVICE_PORT)/index.html"]
-  
+
 ```
-**Note:** As we don't set limits and request specifically in the deployment the default LimitRange will apply.
+**Note:** As we don't set limits and request specifically in the deployment the default LimitRange will apply. Run `oc describe LimitRange/default-limits` to see what is set as defaults.
 
-From the web console if you change to developer view and navigate to the Monitoring tab select your nginx deployment. You should see the load-test pod traffic increasing metrics for our pod.
+From the web console if you change to developer view and navigate to the Observe tab select your nginx deployment. You should see the load-test pod traffic increasing CPU and Memory usage metrics for nginx workload.
```
Comment (mtspn) [inline openshift-201/resource-mgmt.md:121]: Just noting that the change of the name of the tab (from Monitoring -> Observe) will potentially impact other areas of these lab instructions, screenshots, and the 101 lab.

**Review by mtspn — CHANGES_REQUESTED** [review]
Thanks for the edits Shelly, looks good! I've just suggested some minor changes.

File: openshift-201/resource-mgmt.md (hunk 3 above)
Comment (ShellyXueHan) [reply openshift-201/resource-mgmt.md:121]: working my way through!

**Review by mtspn — APPROVED** [review]

File: openshift-201/resource-mgmt.md (hunk 3 above)
Comment (mtspn) [reply openshift-201/resource-mgmt.md:121]: I've gone in to the 101 lab and updated all text references to the 'monitoring' button and changed it there too.

**Pull request merged**